4-star-nosed_mole/api_definitions/rest/drs3.yaml

```diff
@@ -0,0 +1,225 @@
+  termsOfService: https://www.ga4gh.org/terms-and-conditions/
+  license:
+    name: Apache 2.0
+    url: https://raw.githubusercontent.com/ga4gh/data-repository-service-schemas/master/LICENSE
+  version: 1.1.0
+servers:
+  - url: /ga4gh/drs/v1
+security:
+  - {}
+  - authToken: []
+tags:
+  - name: DataRepositoryService
+paths:
+  /objects/{object_id}:
+    get:
+      tags:
+        - DataRepositoryService
+      summary: Get info about a `DrsObject`.
+      description:
+        Returns object metadata, and a list of access methods that can
+        be used to fetch object bytes.
+      operationId: GetObject
+      parameters:
+        - name: object_id
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: The `DrsObject` was found successfully.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DrsObject"
+        202:
+          description: |
+            The operation is delayed and will continue asynchronously. The client should retry this same request after the delay specified by Retry-After header.
+          headers:
+            Retry-After:
+              description: |
+                Delay in seconds. The client should retry this same request after waiting for this duration. To simplify client response processing, this must be an integral relative time in seconds. This value SHOULD represent the minimum duration the client should wait before attempting the operation again with a reasonable expectation of success. When it is not feasible for the server to determine the actual expected delay, the server may return a brief, fixed value instead.
+              schema:
+                type: integer
+                format: int64
+          content: {}
+        400:
+          description: The request is malformed.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        401:
+          description: The request is unauthorized.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        403:
+          description: The requester is not authorized to perform this action.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        404:
+          description: The requested `DrsObject` wasn't found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        500:
+          description: An unexpected error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-swagger-router-controller: ga4gh.drs.server
```

Comment:
What's the role of `x-swagger-router-controller`?

Reply:
That was also inherited from the original DRS spec.

But I don't think we will have immediate use for it:
https://github.com/swagger-api/swagger-node/blob/master/docs/controllers.md

Will remove it.